src/thorlabs_cube/driver/tcube/tpz.py

```diff
@@ -1,20 +1,26 @@
 import struct as st
+from typing import Optional, Tuple
+
+import asyncserial
 
 from thorlabs_cube.driver.base import _Cube
 from thorlabs_cube.driver.message import MGMSG, Message, MsgError
 
+_CHANNEL = 0x01
+_RESERVED = 0x00
```

Comment:
```suggestion
_CHANNEL: int = 0x01
_RESERVED: int = 0x00
```
Prefer type hints for constants.